srcpkgs/darkhttpd/files/darkhttpd/run

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
 [ -r conf ] && . ./conf
+[ -d "/var/log/darkhttpd" ] || mkdir /var/log/darkhttpd
```

Comment:
why is this needed if it's already in make_dirs?

Reply:
Good grief you mentioned make_dirs because I have not noticed it there.